safe-global/web-core
# change: adjust safe token widget

## What does this PR resolve
- format amount using two decimals
- link to safe app

## How to test it
1. open the app with a safe which holds safe tokens:
i.e. `rin:0x979774d85274A5F63C85786aC4Fa54B9A4f391c2`
2. observe the format of the safe tokens in the header
3. observe the new logo
4. click on the widget 
5. observe the safe app opening

## Analytics changes
None

## Screenshots
![localhost_3000_rin_0x4f99325f843524dbedc853dE85D6F184F60C250A_apps_appUrl=https%3A%2F%2Fsafe-claiming-app pages dev%2F](https://user-images.githubusercontent.com/2670790/188140582-bf3b84c9-5afc-4244-aa6a-c5520addc434.png)
